fableous-fe/src/containers/HubCanvasPage.tsx

```diff
@@ -340,97 +354,214 @@ export default function HubCanvasPage() {
         </>
       )}
       {hubState === HubState.SessionForm && (
-        <Formik
-          initialValues={
-            {
-              title: "",
-              description: "",
-              pages: 1,
-            } as Story
-          }
-          validationSchema={yup.object({
-            title: yup.string().required("required"),
-            description: yup.string().required("required"),
-            pages: yup.number().positive("must be positive"),
-          })}
-          onSubmit={handleCreateSession}
-        >
-          {(formik) => (
-            <form onSubmit={formik.handleSubmit}>
-              <div>
-                <FormikTextField
-                  formik={formik}
-                  name="title"
-                  label="Title"
-                  overrides={{
-                    variant: "outlined",
-                    disabled: postLoading,
-                    className: "mb-4",
+        <Grid item xs={12} sm={8} md={6} lg={4}>
+          <Card>
+            <Formik
+              initialValues={
+                {
+                  title: "",
+                  description: "",
+                  pages: 1,
+                } as Story
+              }
+              validationSchema={yup.object({
+                title: yup.string().required("Title required"),
+                description: yup.string().required("Description required"),
+                pages: yup
+                  .number()
+                  .required("Number of pages required")
+                  .integer("Invalid numbergit")
+                  .positive("Must have at least one page")
+                  .lessThan(21, "Maximum is 20 pages"),
+              })}
+              onSubmit={handleCreateSession}
+            >
+              {(formik) => (
+                <form onSubmit={formik.handleSubmit} autoComplete="off">
+                  <CardContent>
+                    <Grid container spacing={2}>
+                      <Grid item xs={12} className="flex-grow flex flex-col">
+                        <FormikTextField
+                          formik={formik}
+                          name="title"
+                          label="Title"
+                          overrides={{
+                            autoFocus: true,
+                            variant: "outlined",
+                            disabled: postLoading,
+                          }}
+                        />
+                      </Grid>
+                      <Grid item xs={6} className="flex-grow flex flex-col">
```

Comment:
do we want to make this and one below it xs={12} and either sm={6} or md={6} ?

Reply:
done